src/test/suite/module.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as assert from "assert";
+import { platform } from "os";
+import { format, getText } from "./format.test";
+
+suite("Test module resolution", function () {
+  this.timeout(10000);
+
+  test("it formats without prettier in package.json", async () => {
+    if (platform() === "win32") {
+      return assert.ok(true, "Skipping test on windows.");
+    }
+    const { actual } = await format("module", "index.js");
+    const expected = await getText("module", "index.result.js");
```

Comment:
Out of curiosity, does something in the test suite prevent this from using the Prettier version built into the extension? I figured a test case like this would actually pass prior to the changes I made as well because of that default.

Reply:
Yeah, good point. I didnt even think about this. I need to change the test to check the version.